src/components/form-elements/Input.jsx

```diff
@@ -122,7 +122,7 @@ export type InputPropsType = {
   /**
    * Additional function to set ref for input
    */
-  setInputRef?: (ref: ?HTMLElement) => mixed,
+  setInputRef?: { current: ?HTMLInputElement } | (ref: ?HTMLElement | ?HTMLInputElement) => mixed,
```

Comment:
did you thought about just passing ref like in FIleHandler? not a function?

Reply:
That's how I'm passing it. But I don't want to break the API, so I left the previous function-way as well